doc/chapters/integration-guidelines/10-systemd-service.rst

```diff
@@ -0,0 +1,22 @@
+systemd.service
+===============
+
+As it is mentioned in :ref:`Getting Started <getting-started>` section the user API is implemented by
+the SoftwareContainer Agent as a D-bus service. The API is used to start, configure, and control
+containers. Thus in order to use SoftwareContainers the agent should be run first. To be able to start
+the agent and watch its process/system logs it would be wise decision to use systemd.service.
```

Comment:
freedesktop calls it systemd.service, thats why i wanted to stick with that wording. (ref:https://www.freedesktop.org/software/systemd/man/systemd.service.html)